tester/src/transactions.rs

```diff
@@ -0,0 +1,92 @@
+                    return Err(Error::InvalidField("inputs"));
+                }
+            }
+
+            let mut outputs = Vec::new();
+
+            if let Some(json_outputs) = transaction.get("outputs") {
+                if let Some(json_outputs) = json_outputs.as_array() {
+                    for output in json_outputs {
+                        if let Some(_dto) = output.get("dto") {
+                        } else if let Some(simple) = output.get("simple") {
+                            let account = if let Some(account) = simple.get("account") {
+                                if let Some(account) = account.as_u64() {
+                                    account as usize
+                                } else {
+                                    return Err(Error::InvalidField("account"));
+                                }
+                            } else {
+                                return Err(Error::MissingField("account"));
+                            };
+
+                            let amount = if let Some(amount) = simple.get("amount") {
+                                if let Some(amount) = amount.as_u64() {
+                                    amount
+                                } else {
+                                    return Err(Error::InvalidField("amount"));
+                                }
+                            } else {
+                                return Err(Error::MissingField("amount"));
+                            };
+
+                            let address =
+                                if let Some(account) = context.account_manager.get_accounts().await?.get(account) {
+                                    account.addresses().await?[0].address().as_ref().clone()
+                                } else {
+                                    return Err(Error::InvalidField("account"));
+                                };
+
+                            // TODO unwrap
+                            let simple_output = BasicOutputBuilder::new_with_amount(amount)
+                                .unwrap()
+                                .add_unlock_condition(UnlockCondition::Address(AddressUnlockCondition::new(address)))
+                                .finish_output(context.protocol_parameters.token_supply())
+                                .unwrap();
+
+                            outputs.push(simple_output);
+                        } else {
+                            return Err(Error::InvalidField("output"));
+                        }
+                    }
+                } else {
+                    return Err(Error::InvalidField("outputs"));
+                }
+            } else {
+                return Err(Error::MissingField("outputs"));
+            }
+
+            let _transaction = context.account_manager.get_accounts().await?[0]
+                .send(outputs, None)
+                .await?;
+
+            time::sleep(Duration::from_secs(10)).await;
```

Comment:
If that's for waiting that the tx gets confirmed, it might be better to use 
```
    if let Some(block_id) = transaction.block_id{
        account
        .retry_until_included(&block_id, None, None)
        .await?;
    }else{
        account.sync(None).await?;
        time::sleep(Duration::from_secs(10)).await;
    }
```

Reply:
Done, thanks